networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/discovery/ConnectionManagerTest.java

```diff
@@ -43,18 +43,19 @@
 
 class ConnectionManagerTest {
```

Comment:
Worth adding a test for discovery peers with no enr fork ID.

Reply:
I believe adding that test to `DiscoveryNetworkTest.java` would be better since the connection manager is agnostic for any Eth2 specific information, and the enr fork id predicates get added at DiscoveryNetwork.